js/app.js

```diff
@@ -0,0 +1,502 @@
+service = angular.module("shoppingcart.services", ["ngResource"]);
+
+service.factory('Order', function($resource) {
+    param = {};
+    var item = $resource(window.location.pathname, param);
+    return item;
+});
+
+var flyp_base = 'https://flyp.me/api/v1';
+
+service.factory('AltcoinNew', function($resource) {
+    var rsc = $resource(flyp_base + '/order/new');
+    return rsc;
+});
+
+service.factory('AltcoinAccept', function($resource) {
+    var rsc = $resource(flyp_base + '/order/accept');
+    return rsc;
+});
+
+service.factory('AltcoinCheck', function($resource) {
+    var rsc = $resource(flyp_base + '/order/check');
+    return rsc;
+});
+
+service.factory('AltcoinInfo', function($resource) {
+    var rsc = $resource(flyp_base + '/order/info');
+    return rsc;
+});
+
+service.factory('AltcoinAddRefund', function($resource) {
+    var rsc = $resource(flyp_base + '/order/addrefund');
+    return rsc;
+});
+
+service.factory('AltcoinLimits', function($resource) {
+    var rsc = $resource(flyp_base + '/order/limits/:coin/BTC', {
+        coin: '@coin'
+    });
+    return rsc;
+});
+
+service.factory('WpAjax', function($resource) {
```

Comment:
Refactor the service name. `WpAjax` will be confusing later, this is basically an internal API call so maybe something referring to that

Reply:
Refactor in 49cf590